packages/office-ui-fabric-react/src/components/Callout/Callout.styles.tsx

```diff
@@ -11,7 +11,7 @@ export function getStyles(props: ICalloutStyleProps): ICalloutStyles {
     root: [
       'ms-Callout',
       {
-        boxShadow: '0 0 5px 0px',
+        boxShadow: '0 0 5px 0px rgba(0,0,0,.4)',
```

Comment:
Noice!

Reply:
yup..design detail down at the callout level will have a BIG impact on all the components that use it